src/components/Post/MoreText.tsx

```diff
@@ -0,0 +1,27 @@
+import { Collapse, CollapseProps, Button, useBoolean } from '@chakra-ui/react';
+
+type MoreTextProps = CollapseProps & {
+  text: string;
+  maxHeight: number;
+};
+
+const MoreText = ({ text, maxHeight, ...props }: MoreTextProps) => {
+  const [toggle, setToggle] = useBoolean(false);
+
+  const handleToggle = () => {
+    setToggle((prev) => !prev);
```

Comment:
```suggestion
    setisToggled.toggle();
```
`useBoolean`훅 안에서 제공된 함수들을 사용해주세요!!